### PR TITLE
Master fix session request method

### PR DESCRIPTION
**Pull request by chaikovskyia**

### CODE REVIEW

File: Model/Api/Client.php [hunk 1]
```diff
@@ -683,7 +683,8 @@ public function getSessionInfo($sessionId, $scopeCode = null)
     {
         $this->scope = $scopeCode;
         $endpoint = $this->getCheckoutApiUri(sprintf('sessions/%s', $sessionId));
-        $request = $this->initRequest($endpoint, $this->getToken())->setBody(null);
+        $request = $this->initRequest($endpoint, $this->getToken())->setBody(null)
```
Comment (andersem) [inline Model/Api/Client.php:686]: Do you need `setBody(null)`? It's not used in `getTransaction`.

File: Model/Dintero.php [hunk 2]
```diff
@@ -100,7 +100,7 @@ class Dintero extends AbstractMethod
      *
      * @var bool
      */
-    protected $_canUseInternal = true;
+    protected $_canUseInternal = false;
```
Comment (andersem) [inline Model/Dintero.php:103]: What does this do?

**Review by andersem — APPROVED** [review]

File: Model/Dintero.php (hunk 2 above)
Comment (chaikovskyia) [reply Model/Dintero.php:103]: it disables the usage of dintero when creating orders from admin panel.

File: Model/Api/Client.php (hunk 1 above)
Comment (chaikovskyia) [reply Model/Api/Client.php]: @andersem , it is not needed, I've removed it in the next commit.

**Review by andersem — APPROVED** [review]

**Pull request merged**